pkg/trtl/honu.go

```diff
@@ -23,14 +24,31 @@ func NewHonuService(s *Server) (*HonuService, error) {
 }
 
 // Get is a unary request to retrieve a value for a key.
-func (h *HonuService) Get(ctx context.Context, in *pb.GetRequest) (out *pb.GetReply, err error) {
+func (h *HonuService) Get(ctx context.Context, in *pb.GetRequest) (*pb.GetReply, error) {
+	var err error
+
+	if len(in.Key) == 0 {
+		log.Error().Msg("missing key in Trtl Get request")
+		return nil, status.Error(codes.InvalidArgument, "key must be provided in Get request")
+	}
+	if in.Options == nil {
+		log.Error().Msg("missing options in Trtl Get request")
+		return nil, status.Error(codes.InvalidArgument, "options must be provided in Get request")
```

Comment:
I'm not sure I agree that options should be required - but perhaps we could revisit and discuss?